packages/itwinui-css/src/table/table.scss

```diff
@@ -201,6 +201,7 @@
       border-left: 1px solid transparent;
       border-right: 1px solid transparent;
       border-bottom: 1px solid var(--iui-color-border);
+      flex-shrink: 0;
```

Comment:
How did you test these changes? How do I reproduce your gifs? (please dont clear the PR description template)

Why does it work and will it have any adverse affect in edge cases like virtualization?

Reply:
How I tested this:
1. Go to [the expandable table Storybook demo](https://itwin.github.io/iTwinUI/react/iframe.html?args=&id=core-table--expandable&viewMode=story).
2. Set `height: 200px;` on `.iui-table`.
3. Try to expand a row and it won't display.

I think the user was asking for a solution to the 3rd GIF in the description which we already offered a solution to.  While looking into the original issue, I discovered this bug.

Reply:
Dont forget changeset.

Reply:
Changeset added.